src/payment/strategies/paypal-commerce/paypal-commerce-request-sender.ts

```diff
@@ -38,4 +38,16 @@ export default class PaypalCommerceRequestSender {
 
         return res.body as OrderData;
     }
+
+    async getOrderStatus(): Promise<OrderStatus> {
+        const url = `/api/storefront/initialization/paypalcommerce`;
+        const headers = {
+            'X-API-INTERNAL': INTERNAL_USE_ONLY,
+            'Content-Type': ContentType.Json,
+        };
+
+        const res = await this._requestSender.get(url, {headers});
+
+        return res.body as OrderStatus;
```

Comment:
instead of doing `as OrderStatus`, you should be able to set the type on line 49
```
const res = await this._requestSender.get< OrderStatus>(url, {headers});
```

Reply:
fixed